### PR TITLE
Urls validation

### PR DESCRIPTION
**Pull request by HemaSAli**
relate to #65

### CODE REVIEW

**Review by ali-7 — APPROVED** [review]

**Review by lubnaabd — APPROVED** [review]

File: public/css/projectWebsite.css [hunk 1]
```diff
@@ -142,6 +141,8 @@ color: #757575;
   border-radius: 5px;
 
   max-width: 100%;
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]:
```css
max-width: 100%;
width: 100%;
```
What r u trying to achieve here?

File: src/views/ProjectsPageWebStite.hbs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    International. We train 16 students per cohort in a full-time, intensive course for 8 weeks with an additional 16 weeks
+    of project-based learning with real-world clients to jumpstart your professional portfolio. The objective is to graduate
+    as full-stack developers who can deploy production-grade software online and secure high-quality jobs with companies
+    or work as freelance developers.</p>
+  <hr>
+</section>
+<section class="client-project">
+  {{#if err}}
+<h3>No Result</h3>
+{{else}}
```
Comment (NouraldinS) [inline src/views/ProjectsPageWebStite.hbs]: Fix messed up indentation.

File: src/views/cohortPageWebsite.hbs [hunk 3]
```diff
@@ -17,47 +17,43 @@
...
+  {{#if communityProjects.length}}
     <div class="proj">
-      {{#each res.projectResults}}
-       <a href="/projects/{{id}}">
+      {{#each communityProjects}}
+       <a href="/project/{{id}}">
         <div class="comp" id="{{cohort_id}}">
-          <img class="projectImg" src="{{img_url}}">
-          <h4>{{name}}</h4>
+          {{!-- <img class="projectImg" src="{{img_url}}"> --}}
```
Comment (NouraldinS) [inline src/views/cohortPageWebsite.hbs]: Remove commented code.

File: src/views/helpers/compare.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module.exports = (firstWord, secondWord, options) => {
+  if (firstWord === secondWord) {
+    return options.fn(this);
+  } return options.inverse(this);
```
Comment (NouraldinS) [inline src/views/helpers/compare.js]: Add newline after curly brace.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

**Review by NouraldinS — APPROVED** [review]

**Pull request merged**